base/util_testing.go

```diff
@@ -64,6 +64,25 @@ func (tb *TestBucket) GetUnderlyingBucket() Bucket {
 	return tb.Bucket
 }
 
+// AsCollectionStore gets the underlying collection store from a bucket for test purposes
+func AsCollectionStore(t testing.TB, bucket Bucket) (*Collection, bool) {
```

Comment:
`CollectionStore` is not what we're returning from this function (other cases return something implementing a *Store interface)

I don't think we need this to be test-specific either - @torcolvin had mentioned needing to write this function.

```suggestion
// AsCollection gets the underlying Collection from a bucket
func AsCollection(bucket Bucket) (*Collection, bool) {
```